_layouts/default.html

```diff
@@ -28,15 +28,15 @@
     <section id="discover" class="components-list">
 
       {% include _components/resources-resolver.html %}
+      {% include _components/security.html %}
       {% include _components/doctrine-events.html %}
       {% include _components/auto-registration.html %}
-      {% include _components/user.html %}
       {% include _components/fixtures-load.html %}
-      {% include _components/security.html %}
+      {% include _components/user.html %}
       {% include _components/view-renderer.html %}
       {% include _components/domain-event.html %}
       {% include _components/url-generation.html %}
-      {% include _components/prototype.html %}
+      <!--{% include _components/prototype.html %}-->
```

Comment:
I guess, twig comment would be more suitable here. As far as I understand, the template is still being included.